.github/workflows/main.yaml

```diff
@@ -0,0 +1,114 @@
+name: CI
+on:
+  push:
+    branches:
+      - 'main'
+      - 'scratch/**'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+
+env:
+  BBI_VERSION: v3.3.0
+
+jobs:
+  check:
+    runs-on: ${{ matrix.config.os }}
+    name: ${{ matrix.config.os }} R ${{ matrix.config.r }} ${{ matrix.config.label }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - os: ubuntu-20.04
+            r: 4.2.3
+          - os: ubuntu-20.04
+            r: 4.3.1
+          - os: ubuntu-latest
+            r: release
```

Comment:
How does `r: release` work? Is this set somewhere else?

Reply:
Yes, it's fed to as the `r-version` input to `r-lib/actions/setup-r@v2` below.

https://github.com/metrumresearchgroup/bbr/blob/72b1d4a12aa0b31d5cc3217040d62981dd6aa188/.github/workflows/main.yaml#L46-L48

https://github.com/r-lib/actions/blob/fbafc3bc4ba114e72680c71e835c59b022606c46/setup-r/action.yml#L1-L10